README.md

```diff
@@ -20,7 +19,6 @@ You will need the following things properly installed on your computer.
 $ git clone git@github.com:heroku/ember-js-getting-started.git
 $ cd ember-js-getting-started
 $ npm install
```

Comment:
could you update the instructions here to use yarn instead of npm? (and probably add yarn as a dependency in the list above)